### ISSUE TITLE
Since 8.3.7 search does not work in browser if there is Vietnamese locale in i18n plugin

### ISSUE THREAD

**Issue by oneumyvakin** [squidfunk/mkdocs-material#7498]
### Bug description

lunrjs has en issue https://github.com/olivernn/lunr.js/issues/324 when working in strict mode ("use strict";)
Since 8.3.7 "use strict"; added to the beginning of search bundle.
This lead to the issue search in blower doesn't work because of JS error

search.07f07601.min.js:8 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'console')
    at Object.warn (search.07f07601.min.js:8:61)
    at t.Pipeline.warnIfFunctionNotRegistered (search.07f07601.min.js:17:393)
    at t.Pipeline.<anonymous> (search.07f07601.min.js:18:319)
    at Array.forEach (<anonymous>)
    at t.Pipeline.add (search.07f07601.min.js:18:288)
    at t.Builder.<anonymous> (lunr.multi.min.js:1:686)
    at t.Builder.use (search.07f07601.min.js:40:3184)
    at t.Builder.<anonymous> (search.07f07601.min.js:40:17975)
    at t (index.ts:192:2)
    at new H (search.07f07601.min.js:40:17829)

This issue happened only if Vietnamese locale exists in i18n plugin

### Related links

- https://github.com/olivernn/lunr.js/issues/324
- Commit https://github.com/squidfunk/mkdocs-material/commit/e86896645bd3d9407adf3711703d2d7f04b00556#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

### Reproduction

[Uploading 9.5.33-vietnamese-search.zip…]()

### Steps to reproduce

1. Define mkdocs:
```
site_name: My Docs

theme:
  name: material

plugins:
  - search
  - i18n:
        languages:
              - locale: vi
                name: Vietnamese
                default: true
                build: yes
                nav_translations:
                  test: Hướng
```
2. mkdocs serve --dev-addr=0.0.0.0:8000
3. Try to search something.
4. Check browsers console.

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for reporting. Please report this to the i18n plugin. We don't officially support this plugin, and without it, Vietnamese search works, so it must be related to the plugin. Closing as out of scope.

**Comment by squidfunk** [comment]
Reopened due to https://github.com/ultrabug/mkdocs-static-i18n/issues/313#issuecomment-2355890746 and fixed in 8925cae63. This still prints a warning, which we'd need to take to Lunr Languages, as I've tried to fix it but to no avail. However, search works now.

**Comment by squidfunk** [comment]
Released as part of [9.5.35](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.35).
